server/historian/packages/historian-base/src/routes/utils.ts

```diff
@@ -118,18 +123,36 @@ export async function createGitService(createArgs: createGitServiceArgs): Promis
 	const customData: ITenantCustomDataExternal = details.customData;
 	const writeToExternalStorage = !!customData?.externalStorageData;
 	const storageUrl = config.get("storageUrl") as string | undefined;
+	const ignoreEphemeralFlag: boolean = config.get("ignoreEphemeralFlag");
 	const maxCacheableSummarySize: number =
 		config.get("restGitService:maxCacheableSummarySize") ?? 1_000_000_000; // default: 1gb
 
-	let isEphemeral = cache ? isEphemeralContainer : false;
+	let isEphemeral: boolean = isEphemeralContainer;
 	if (!ignoreEphemeralFlag) {
-		if (isEphemeralContainer !== undefined) {
-			await cache?.set(`isEphemeral:${documentId}`, isEphemeralContainer);
+		const isEphemeralKey: string = `isEphemeralContainer:${documentId}`;
+		if (isEphemeral !== undefined && isEphemeral !== null) {
```

Comment:
isn't this equivalent to if (isEphemeral )?

Reply:
ah got it..thanks for explaining..you it be simpler to say `typeof(isEphemeral) === "boolean"`. That way we might not have to do the eqeqeq disabling



https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABBAFgUwgawJIGcCiADugLZoBOAhgDYAUMBxaZV1AXIgEZxzVqVIAPojAhq1RMPAATNMBhg00gJQduvfkgDeAKESJyaKCHJIoAT0Jo4weo1IUayxAF43iAETq+AjzoC+OjoA9MGIAKq4lADmaDoQCLhQBmi4YlAAjK7IiRoAdNRw0bSoGDj2zI50UOQgaMrKANyIoYg1dfGJyYZp1FAATNkJYLj5hcWlWHhEDqy0wDS49U0tYe1xw0kpvVAAzEO5fAVFJehTFSw0tKLiDc2tC9RLiJ0j3anpACwHI2Mnk+UZpU5jI5AolHdVohHksgA

Reply:
Oh yeah, good idea, I like this better